sigstore/_cli.py

```diff
@@ -227,7 +224,7 @@ def _parser() -> argparse.ArgumentParser:
         "-v",
         "--verbose",
         action="count",
-        default=argparse.SUPPRESS,
+        default=0,
```

Comment:
IIRC correctly this will break "-v" at some location of the command as now every parser will set verbose value (even if a previous parser set a non-zero value).... maybe it was `sigstore -v cmd` that breaks this way (see f5dbd71d)

or does your overwriting of `args.verbose` deal with this issue?

Reply:
Ah yeah, I had forgotten why we used `SUPPRESS` 😅. I think this does indeed break that change, so I'll need to re-think this.

Reply:
Hmm, I did some quick tests, and this still works as expected, albeit the `sigstore -v subcommand -v` pattern is still only `verbose=1` (same as it was with https://github.com/sigstore/sigstore-python/commit/f5dbd71df7afb507ea901623173f2f6e3935fb5a). So I _think_ this is fine to merge as-is.

Things I tested:

```
sigstore -v sign   # works, sets DEBUG on sigstore logger
sigstore -vv sign  # works, sets DEBUG on root logger
sigstore sign -v   # works, sets DEBUG on sigstore logger
sigstore sign -vv  # works, sets DEBUG on root logger
```